Source/Core/DolphinQt/Config/Graphics/BalloonTip.h

```diff
@@ -0,0 +1,38 @@
+// Copyright 2020 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <QWidget>
+
+#include <QPixmap>
+
+class BalloonTip : public QWidget
+{
+  Q_OBJECT
+public:
+  static void showBalloon(const QIcon& icon, const QString& title, const QString& msg,
+                          const QPoint& pos, int timeout, bool showArrow = true,
+                          QColor background = QColor(0xff, 0xff, 0xe1), QColor text = Qt::black);
+  static void hideBalloon();
+  static bool isBalloonVisible();
+  static void updateBalloonPosition(const QPoint& pos);
+
+private:
+  BalloonTip(const QIcon& icon, const QString& title, const QString& msg, QColor background,
+             QColor text);
+  ~BalloonTip();
+  void balloon(const QPoint&, int, bool);
+
+protected:
+  void paintEvent(QPaintEvent*) override;
+  void resizeEvent(QResizeEvent*) override;
+  void mousePressEvent(QMouseEvent* e) override;
+  void timerEvent(QTimerEvent* e) override;
+
+private:
+  QPixmap pixmap;
+  int timerId;
+  bool showArrow;
```

Comment:
```suggestion
  bool m_show_arrow = true;
```

Would allow the initializer to be removed from the constructor initializer list